ert_shared/_doc_utils/ert_jobs.py

```diff
@@ -138,11 +139,15 @@ def _divide_into_categories(jobs):
                 _ErtDocumentation._CATEGORY_DEFAULT,
             )
 
-            main_category = category.split(".")[0]
-            if main_category not in categories:
-                categories[main_category] = list()
+            split_categories = category.split(".")
+            if len(split_categories) > 1:
+                main_category, sub_category = split_categories[0:2]
```

Comment:
Are we not going to warn or error if someone injected a third category?

Reply:
The idea was the the categories are hierarchical, so one could very well inject a third of fourth sub-category to make it even more specific. However, the grouping in the documentation will only group by the two top level categories :slightly_smiling_face: